src/bots/mascot-bot.js

```diff
@@ -39,6 +46,20 @@ class MascotBot extends SlackBot {
     });
   }
 
+  setTopic(channel, topic, isPublic = true) {
```

Comment:
I still support having a function that can match channel names to IDs to be consistent with how every other function works. Barring that, at least name the parameter channelId.